contracts/ERC827Token.sol

```diff
@@ -29,7 +29,7 @@ contract SmartToken is StandardToken {
 
      @return true if the call function was executed successfully
```

Comment:
I don't think this function (or any of the others in this contract) will ever return false since they use `require` to check that the call to `_spender` was successful.

Reply:
It wont, but it is done like that to follow the same way ERC20 functions are implemented, the `StandardToken` of zeppelin only returns true at the end of the function too.